test/bft_test_exceptions.py

```diff
@@ -14,14 +14,29 @@ class Error(Exception):
     """Base class for exceptions in this module."""
     pass
 
+##
+## Exceptions for bft_tester
+##
 class AlreadyRunningError(Error):
     def __init__(self, replica):
         self.replica = replica
+        j
```

Comment:
Is this `j` a typo?

Reply:
Yes indeed! Good catch. That would have been a fun bug during a test run if that exception was actually raised.

Reply:
I pushed up another small commit to fix this an another bug.